cupy/padding/pad.py

```diff
@@ -1,321 +1,788 @@
-import numpy
-import six
+from __future__ import division, absolute_import, print_function
```

Comment:
CuPy will drop support for Python2 from the next version, so we don't have to use it.